src/test/amount_tests.cpp

```diff
@@ -86,6 +90,31 @@ BOOST_AUTO_TEST_CASE(GetFeeTest)
     CFeeRate(MAX_MONEY, std::numeric_limits<size_t>::max() >> 1).GetFeePerK();
 }
 
+BOOST_AUTO_TEST_CASE(CFeeRateNamedConstructorsTest)
+{
+    // Test CFeerate(CAmount fee_rate, FeeEstimatemode mode) constructor
+    // with BTC/kvB, returns same values as CFeeRate(amount) or CFeeRate(amount, 1000)
+    BOOST_CHECK(CFeeRate::FromBtcKb(CAmount(-1)) == CFeeRate(-1));
+    BOOST_CHECK(CFeeRate::FromBtcKb(CAmount(-1)) == CFeeRate(-1, 1000));
```

Comment:
in afba3e188fcb14236324a0d1355445a99b85d155:

Aren't we already testing somewhere that `CFeeRate(-1) == CFeeRate(-1, 1000)`? Then I think we could save one of those and some other similar lines following.

Reply:
Dropped that line and three other ones from this test.